docs/topics/logging.rst

```diff
@@ -193,6 +193,15 @@ to override some of the Scrapy settings regarding logging.
     Module `logging.handlers <https://docs.python.org/2/library/logging.handlers.html>`_
         Further documentation on available handlers
 
+Custom Log Formats
+-------------------
+
+Custom log format can be set for different actions by extending :class:`~scrapy.logformatter.LogFormatter` class
+and making :setting:`LOG_FORMATTER` inside ``settings.py`` point to your new class.
```

Comment:
Settings can be defined outside `settings.py`, for example [within spiders](https://doc.scrapy.org/en/latest/topics/settings.html#settings-per-spider), so here less is more :smile: 

```suggestion
and making :setting:`LOG_FORMATTER` point to your new class.
```